### PR TITLE
[ticket/15676] Display privacy notice in footer & allow overriding U_PRIVACY

### PR DESCRIPTION
**Pull request by marc1706**
PHPBB3-15676

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15676

### CODE REVIEW

**Comment by marc1706** [comment]
Look of links & adjusted line-height (slightly less crammed than before):
image

**Comment by marc1706** [comment]
@hanakin Would be nice to have some input of whether the line-height and link change is ok with you as well ;)

File: phpBB/styles/prosilver/template/overall_footer.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
 		{CREDIT_LINE}
 		<!-- IF TRANSLATION_INFO --><br />{TRANSLATION_INFO}<!-- ENDIF -->
 		<!-- EVENT overall_footer_copyright_append -->
+		<br><a href="{U_PRIVACY}" title="{L_PRIVACY}" role="menuitem"><span>{L_PRIVACY}</span></a>&nbsp;&vert;&nbsp;<a href="{U_TERMS_USE}" title="{L_TERMS_USE}" role="menuitem"><span>{L_TERMS_USE}</span></a>
```
Comment (hanakin) [inline phpBB/styles/prosilver/template/overall_footer.html]: just use a paragraph tag we should not use br its not style/themeable.

did we already have these lang strings? I would just say "privacy" and "terms" this is the common representation of these, also why not use twig?

Comment (marc1706) [reply phpBB/styles/prosilver/template/overall_footer.html]: These already existed. We could also add these shorter versions as I was thinking the same thing.

Comment (marc1706) [reply phpBB/styles/prosilver/template/overall_footer.html]: Fwiw, I wouldn't change this to a <p> right now. Using <p> adds a whole new set of issues like weird line-height, margin-bottom, etc. As this fits with what we're currently using I'd keep it like that and just change the links to have shorter titels.

File: phpBB/styles/prosilver/theme/common.css [hunk 2]
```diff
@@ -981,8 +981,9 @@ fieldset.fields1 dl.pmlist dd.recipients {
 /* Miscellaneous styles
 ---------------------------------------- */
 .copyright {
-	padding: 5px;
+	line-height: 16px;
```
Comment (hanakin) [inline phpBB/styles/prosilver/theme/common.css]: should be line-height: 1.6

File: phpBB/styles/prosilver/template/overall_footer.html [hunk 3]
```diff
@@ -11,6 +11,7 @@
 		{CREDIT_LINE}
 		<!-- IF TRANSLATION_INFO --><br />{TRANSLATION_INFO}<!-- ENDIF -->
 		<!-- EVENT overall_footer_copyright_append -->
+		<br><a href="{{ U_PRIVACY }}" title="{{ lang('PRIVACY_LINK') }}" role="menuitem"><span>{{ lang('PRIVACY_LINK') }}</span></a>&nbsp;&vert;&nbsp;<a href="{{ U_TERMS_USE }}" title="{{ lang('TERMS_LINK') }}" role="menuitem"><span>{{ lang('TERMS_LINK') }}</span></a>
```
Comment (hanakin) [inline phpBB/styles/prosilver/template/overall_footer.html]: personally I would rather rework/like to see something like this

```html
<div class="copyright">
    <p class="footer-row">
		<span class="footer-copyright">Powered by <a class="footer-brand" href="https://www.phpbb.com/">phpBB</a>® Forum Software © phpBB Limited</span>
	</p><p class="footer-row">
	<a class="footer-link" href="{{ U_PRIVACY }}" title="{{ lang('PRIVACY_LINK') }}" role="menuitem">
		<span class="footer-link-text">Privacy</span>
	</a>
    |
	<a class="footer-link" href="{{ U_TERMS_USE }}" title="{{ lang('TERMS_LINK') }}" role="menuitem">
		<span class="footer-link-text">Terms</span>
	</a>
</p>
	<p class="footer-row">
        <span class="footer-info" title="SQL time: 0.022s / PHP time: 0.108s">Time: 0.130s</span> | <span class="footer-info" title="Cached: 2">Queries: 13</span> | <span class="footer-info">Peak Memory Usage: 3.7 MiB</span>
    </p>
</div>
```

```css
.copyright {
     font-size: 10px;
    text-align: center;
    padding: 10px;
}

.footer-row {
    font-size: 10px;
    line-height: 1.8;
    margin: 0;
}
```

screenshot 2018-05-31 17 59 37

Comment (hanakin) [reply phpBB/styles/prosilver/template/overall_footer.html]: this gives far more control/theme custimizability

Comment (hanakin) [reply phpBB/styles/prosilver/template/overall_footer.html]: if not then at least recommend remove html chars these are no longer required...and just looks weird

**Comment by marc1706** [comment]
Adjusted as requested:
image

**Pull request merged**